GameData/RealismOverhaul/RO_SuggestedMods/Squad/RO_Squad_Utility.cfg

```diff
@@ -439,6 +439,60 @@
 		@resourceAmount = 0.015
 	}
 }
+@PART[spotLight1_v2]:FOR[RealismOverhaul]
+{
+	%RSSROConfig = True
+	@mass = 0.0035
+	@MODULE[ModuleLight]
+	{
+		@resourceAmount = 0.03
+	}
+}
+@PART[spotLight2_v2]:FOR[RealismOverhaul]
+{
+	%RSSROConfig = True
+	@mass = 0.0035
+	@MODULE[ModuleLight]
+	{
+		@resourceAmount = 0.015
+	}
+}
```

Comment:
Mind just editing the patch to the original part to also patch this one, rather than duplicating the patch in its entirety?  Makes for easier maintenance later: no chance of changing one but not the other.

Reply:
Sure, will do.